src/main/java/kitchenpos/repository/OrderRepository.java

```diff
@@ -0,0 +1,27 @@
+package kitchenpos.repository;
+
+import java.util.List;
+import kitchenpos.domain.Order;
+import kitchenpos.domain.OrderStatus;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface OrderRepository extends JpaRepository<Order, Long> {
+
+    @Query("SELECT CASE WHEN COUNT(*) > 0 THEN TRUE ELSE FALSE END "
+            + "FROM Orders o "
+            + "WHERE o.orderTableId = (:orderTableId) AND o.orderStatus IN (:orderStatuses)")
+    boolean existsByOrderTableIdAndOrderStatusIn(
+            @Param("orderTableId") Long orderTableId,
+            @Param("orderStatuses") List<OrderStatus> orderStatuses
+    );
```

Comment:
쿼리가 명시적으로 표현되어 있어 역할을 파악하기 용이하네요!
Spring data jpa에서 제공하는 `method query`를 사용해도 좋아보이네요!
https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#repositories.query-methods.details

아래처럼 수정 할 수 있어요

```suggestion
    boolean existsByOrderTableIdAndOrderStatusIn(
            Long orderTableId,
            List<OrderStatus> orderStatuses
    );
```

Reply:
JPA 진짜 멋쟁이네요!
남겨주신 공식문서도 감사합니다. 읽다보니 [어떤 키워드들을 지원해주는지](https://docs.spring.io/spring-data/jpa/docs/current/reference/html/#appendix.query.method.predicate) 상세하게 나와있는 것을 발견했어요. 앞으로 JPA 를 사용할 때 편리하겠네요 👍